TestComponentCSharp/TestComponentCSharp.idl

```diff
@@ -341,6 +341,31 @@ namespace TestComponentCSharp
         void Observe(Microsoft.UI.Xaml.Interop.IBindableObservableVector vector);
     }
 
+    //[default_interface]
+    //runtimeclass IBindableVectorTest : Microsoft.UI.Xaml.Interop.IBindableVector
+    //{
+    //    //IBindableVectorTest();
+    //}
+
+    [default_interface]
+    runtimeclass IBindableIteratorTest : Microsoft.UI.Xaml.Interop.IBindableIterator
+    {
+        IBindableIteratorTest();
+    }
+
+    [default_interface]
```

Comment:
shouldn't this be in TestWinRT.TestComponent?

Reply:
Yup. Actually, I think we can get rid of it now because I think I added the tests for IDictionary APIs in the existing Dictionary Test in TestWinRT